packages/icon-button/src/icon-button-base.ts

```diff
@@ -83,9 +100,14 @@ export class IconButtonBase extends BaseElement {
         aria-hidden="true"
         aria-label="${this.label}"
         ?disabled="${this.disabled}">
-        <i class="material-icons mdc-icon-button__icon">${this.offIcon}</i>
-        <i class="material-icons mdc-icon-button__icon mdc-icon-button__icon--on">${
-        this.icon}</i>
+        <span class="mdc-icon-button__icon">
+          <slot name="offIcon" @slotchange="${
+        this.calculateShouldToggle}"><i class="material-icons">${
```

Comment:
Maybe if you add a newline before the `<i>` then clang won't try and split on the expression

Reply:
Yep, that works!